chart/templates/configmap.yaml

```
@@ -5,6 +5,11 @@ metadata:
   labels:
     {{- include "chart.labels" . | nindent 4 }}
 data:
+  HTTP_PROXY: "http://hx-wwwcache.ebi.ac.uk:3128"
+  HTTPS_PROXY: "http://hx-wwwcache.ebi.ac.uk:3128"
+  http_proxy: "http://hx-wwwcache.ebi.ac.uk:3128"
+  https_proxy: "http://hx-wwwcache.ebi.ac.uk:3128"
```

The values are identical. One config map key is enough here.
```suggestion
  HX_PROXY: "http://hx-wwwcache.ebi.ac.uk:3128"
```